R/dm.R

```diff
@@ -701,6 +701,27 @@ as.list.zoomed_dm <- function(x, ...) {
   as.list(tbl_zoomed(x))
 }
 
+#' Get a glimpse of `dm` object
```

Comment:
```suggestion
#' Get a glimpse of your `dm` object
```
Alternatively "of a `dm` object"?